packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -1303,6 +1303,103 @@ test('adjust balances', async t => {
   );
 });
 
+test('adjust balances - withdraw RUN', async t => {
```

Comment:
much of what's being tested here is tested elsewhere. I don't suggest deleting but I suspect we'd benefit from being more systematic in what gets tested where and how.

Reply:
Hmm.  From 1306 to 1363, it's setting up an overcollateralized loan. From that point on it's testing the new functionality of withdrawing RUN without changing collateral, which was previously untested.  I'll drop the assertions in the earlier section.